CONTRIBUTING.md

```diff
@@ -0,0 +1,232 @@
+# GitHub Conventions Guide
+
+*A quick reference for our team's GitHub workflow. Read time: ~5 minutes*
+
+---
+
+## Typical Workflow
+
+**Day-to-day process:**
+
+1. **Pick a task** - Look at open issues, assign one to yourself
+2. **Create branch** - `git checkout -b feature/task-name`
```

Comment:
1. Working copy should be clean first.
2. Might be good to mention VS Code workflow.